react/components/FilterSidebar.js

```diff
@@ -74,21 +83,40 @@ const FilterSidebar = ({
     setFilterOperations([])
   }
 
+  const isCategoryDepartmentCluster = term => {
+    return (
+      term === MAP_CATEGORY_CHAR ||
+      term === PRODUCT_CLUSTER_IDS ||
+      term === CATEGORY ||
+      term === DEPARTMENT
+    )
+  }
```

Comment:
That is a pure function. We can move this to be defined outside of the component so we don't create it a new function on every render.

Also, the name and the parameter name are not clear. 

You could do something like

```js
const SEARCH_PARAMS = [MAP_CATEGORY_CHAR, PRODUCT_CLUSTER_IDS, CATEGORY, DEPARTMENT]

const tanana = term => SEARCH_PARAMS.includes(term)
```